.github/workflows/go.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   build:
     strategy:
       matrix:
-        go: [ 1.21.x, 1.22.x, 1.23.x ]
```

Comment:
https://github.com/go-kratos/kratos/blob/5d532fe0cd1ee6f1af1d588dc6d3976f36c36799/go.mod#L3

We can update go version in the ```go.mod```.